### PR TITLE
remove redis 🎉

### PR DESCRIPTION
**Pull request by dwradcliffe**
This will remove redis code and supporting gems. Now that we are pulling stats from postgresql we no longer need this code.

r: @evanphx @indirect @ktheory @arthurnn @skottler

### CODE REVIEW

**Comment by ktheory** [comment]
LGTM. 👍 Great work, @dwradcliffe! 💪 💯

**Comment by simi** [comment]
Nice work! :heart:

**Comment by skottler** [comment]
Yissss. This is so dope, nice work @dwradcliffe.

👍

File: db/migrate/20100817182653_add_version_hash_and_deps_list_to_redis.rb [hunk 1]
```diff
@@ -1,26 +1,9 @@
...
+    # NOOP because we removed redis
   end
 
   def self.down
-    [Redis.current.keys('v:*') + Redis.current.keys('rd:*') + Redis.current.keys('dd:*')].flatten.each do |key|
-      Redis.current.del(key)
-    end
+    # NOOP because we removed redis
   end
 end
```
Comment (arthurnn) [inline db/migrate/20100817182653_add_version_hash_and_deps_list_to_redis.rb]: maybe we should remove this file?

File: test/unit/gem_download_test.rb [hunk 2]
```diff
@@ -114,12 +114,12 @@ def test_update_the_count_atomically
...
-    GemDownload.increment(@version_3.full_name)
-    GemDownload.increment(@version_1.full_name)
-    3.times { GemDownload.increment(@version_3.full_name) }
-    2.times { GemDownload.increment(@version_2.full_name) }
+    GemDownload.increment(1, rubygem_id: @rubygem_1, version_id: @version_1.id)
+    GemDownload.increment(1, rubygem_id: @rubygem_1, version_id: @version_2.id)
+    GemDownload.increment(1, rubygem_id: @rubygem_2, version_id: @version_3.id)
+    GemDownload.increment(1, rubygem_id: @rubygem_3, version_id: @version_1.id)
+    3.times { GemDownload.increment(1, rubygem_id: @rubygem_2, version_id: @version_3.id) }
+    2.times { GemDownload.increment(1, rubygem_id: @rubygem_1, version_id: @version_2.id) }
```
Comment (arthurnn) [inline test/unit/gem_download_test.rb]: why you had to change these?

**Comment by arthurnn** [comment]
This is 🎉

File: test/unit/gem_download_test.rb (hunk 2 above)
Comment (dwradcliffe) [inline test/unit/gem_download_test.rb]: The method signature for these methods had changed. This test is skipped but still seemed better to fix it.

Comment (simi) [inline test/unit/gem_download_test.rb]: https://github.com/rubygems/rubygems.org/commit/4d9067b8e4b31e1378ad9843e549b236b3746c00

```ruby
GemDownload.increment(@version_1.full_name)
# => became
GemDownload.bulk_update([[@version_1.full_name, 1]])
```

Comment (arthurnn) [inline test/unit/gem_download_test.rb]: I changed that signature again on a local branch, will PR soon. and ask for review.

**Pull request merged**